### ISSUE TITLE
heroku-deploy should provide a way to control name of Dockerfile

### ISSUE THREAD

**Issue by chaowenGUO** [AkhileshNS/heroku-deploy#81]
heroku-deploy should provide a way to control name of Dockerfile and context. Now heroku-deploy just recognize the Dockerfile in the appdir path. For some reason, I have to customize different Dockerfile for different running environment. So please provide a way to set the path to Dockerfile.

**Comment by thisismydesign** [comment]
Yes please! having separate Dockerfiles for dev and prod is a quite common use case. I want to use `Dockerfile.prod`.
